ci/tests/puppeteer/scenarios/db-postgres-load/script.json

```diff
@@ -0,0 +1,14 @@
+{
+  "dependencies": "core,jpa-ticket-registry",
+  "properties": [
+    "--cas.jdbc.show-sql=true",
+    "--cas.ticket.registry.jpa.driver-class=org.postgresql.Driver",
```

Comment:
I think it would be good to turn off the audit engine for the test. `--cas.audit.engine.enabled=false`. It's going to produce a lot of noise without this.